src/components/pages/doc/post/post.jsx

```diff
@@ -52,84 +53,109 @@ const Post = ({
   currentSlug,
   fileOriginPath,
   tableOfContents,
-}) => (
-  <>
-    <div
-      className={clsx(
-        'flex flex-col lg:ml-0 lg:pt-0 md:mx-auto md:pb-[70px] sm:pb-8',
-        isUseCase
-          ? 'col-span-6 col-start-4 -mx-10 2xl:col-span-7 2xl:col-start-3 2xl:mx-0 xl:col-span-10 xl:col-start-2'
-          : 'col-span-7 col-start-3 -ml-6 max-w-[832px] 3xl:col-span-8 3xl:col-start-2 3xl:ml-0 2xl:col-span-8 2xl:col-start-1 lg:max-w-none'
-      )}
-    >
-      {breadcrumbs.length > 0 && (
-        <Breadcrumbs
-          breadcrumbs={breadcrumbs}
-          currentSlug={currentSlug}
-          isPostgresPost={isPostgres}
-        />
-      )}
-      {isChangelog ? (
-        <Changelog currentSlug={currentSlug} items={changelogPosts} />
-      ) : (
-        <article>
-          <h1
-            className={clsx(
-              isUseCase
-                ? 'text-[56px] font-semibold leading-dense tracking-tighter xl:text-5xl lg:text-4xl md:text-[28px] md:leading-tight'
-                : 'font-title text-[36px] font-medium leading-tight tracking-tighter xl:text-3xl',
-              tag && 'inline'
-            )}
-          >
-            {title}
-          </h1>
-          {tag && <Tag className="relative -top-1.5 ml-3 inline" label={tag} />}
-          {subtitle && (
-            <p className="my-2 text-xl leading-tight text-gray-new-40 dark:text-gray-new-80">
-              {subtitle}
-            </p>
-          )}
-          <Content
-            className="mt-5"
-            content={content}
-            isUseCase={isUseCase}
-            isPostgres={isPostgres}
+}) => {
+  const isMigratePost = breadcrumbs.some((breadcrumb) =>
+    breadcrumb.title.includes('Migrate to Neon')
```

Comment:
@vannyle will not work for the index page - [Migrate to Neon](https://neon-next-git-migrate-modal-neondatabase.vercel.app/docs/import/migrate-intro) because it doesn't have a slug in breadcrumbs